compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -244,7 +249,14 @@ internal actual class ComposeWindow : UIViewController {
         val width = size.useContents { width } * scale
         val height = size.useContents { height } * scale
         layer.setSize(width.roundToInt(), height.roundToInt())
-        layer.layer.needRedraw()
+        layer.layer.needRedraw() // TODO: remove? the following block should be enough
+        withTransitionCoordinator.animateAlongsideTransition(animation = null) {
+            // Docs: https://developer.apple.com/documentation/uikit/uiviewcontrollertransitioncoordinator/1619295-animatealongsidetransition
```

Comment:
@dima-avdeev-jb JFYI: I added this comment to your change.

P.S.
Interestinlgy, when I tried to use animation callback, it's only invoked once with `percentComplete  = 0`( I expected it would run more than 1 frame). Perhaps, there's some nuances in using it.